test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,53 @@
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+local module = require('testbase')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+require('user_modules/AppTypes')
+
+--- Preconditions
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:DeleteLogsFileAndPolicyTable()
```

Comment:
@LitvinenkoIra, it will be better to use separate functions:
commonSteps:DeletePolicyTable()
commonSteps:DeleteLogsFiles()
as you will delete files with different usage.